### PR TITLE
SWF-4257: Create Feature Harmonize cache

### PR DESCRIPTION
**Pull request by mhbensghaier**

### CODE REVIEW

File: git-workflow/createFB.sh [hunk 1]
```diff
@@ -18,20 +18,20 @@ GATEIN_DEP_TARGET_VERSION=1.5.x-$BRANCH-SNAPSHOT
 ADDON_ES_EMBED_ORIGIN_VERSION=2.0.x-SNAPSHOT
 ADDON_ES_EMBED_TARGET_VERSION=2.0.x-$BRANCH-SNAPSHOT
 # Add-on eXo Answers
-ADDON_ANSWERS_ORIGIN_VERSION=1.3.x-SNAPSHOT
-ADDON_ANSWERS_TARGET_VERSION=1.3.x-$BRANCH-SNAPSHOT
+ADDON_ANSWERS_ORIGIN_VERSION=2.0.x-SNAPSHOT
+ADDON_ANSWERS_TARGET_VERSION=2.0.x-$BRANCH-SNAPSHOT
```
Comment (vsellier) [inline git-workflow/createFB.sh]: should be 2.1.x-... for both

File: git-workflow/createFB.sh [hunk 2]
```diff
@@ -18,20 +18,20 @@ GATEIN_DEP_TARGET_VERSION=1.5.x-$BRANCH-SNAPSHOT
 ADDON_ES_EMBED_ORIGIN_VERSION=2.0.x-SNAPSHOT
 ADDON_ES_EMBED_TARGET_VERSION=2.0.x-$BRANCH-SNAPSHOT
```
Comment (vsellier) [inline git-workflow/createFB.sh]: should be 2.1.x-... for both

File: git-workflow/createFB.sh [hunk 3]
```diff
@@ -18,20 +18,20 @@ GATEIN_DEP_TARGET_VERSION=1.5.x-$BRANCH-SNAPSHOT
...
 # Add-on eXo Answers
-ADDON_ANSWERS_ORIGIN_VERSION=1.3.x-SNAPSHOT
-ADDON_ANSWERS_TARGET_VERSION=1.3.x-$BRANCH-SNAPSHOT
+ADDON_ANSWERS_ORIGIN_VERSION=2.0.x-SNAPSHOT
+ADDON_ANSWERS_TARGET_VERSION=2.0.x-$BRANCH-SNAPSHOT
 # Add-on eXo Chat
-ADDON_CHAT_ORIGIN_VERSION=1.6.x-SNAPSHOT
-ADDON_CHAT_TARGET_VERSION=1.6.x-$BRANCH-SNAPSHOT
+ADDON_CHAT_ORIGIN_VERSION=2.0.x-SNAPSHOT
+ADDON_CHAT_TARGET_VERSION=2.0.x-$BRANCH-SNAPSHOT
```
Comment (vsellier) [inline git-workflow/createFB.sh]: should be 2.1.x-... for both

File: git-workflow/createFB.sh [hunk 4]
```diff
@@ -18,20 +18,20 @@ GATEIN_DEP_TARGET_VERSION=1.5.x-$BRANCH-SNAPSHOT
...
 # Add-on eXo Task
-ADDON_TASK_ORIGIN_VERSION=1.3.x-SNAPSHOT
-ADDON_TASK_TARGET_VERSION=1.3.x-$BRANCH-SNAPSHOT
+ADDON_TASK_ORIGIN_VERSION=2.0.x-SNAPSHOT
+ADDON_TASK_TARGET_VERSION=2.0.x-$BRANCH-SNAPSHOT
```
Comment (vsellier) [inline git-workflow/createFB.sh]: should be 2.1.x-... for both

File: git-workflow/createFB.sh [hunk 5]
```diff
@@ -18,20 +18,20 @@ GATEIN_DEP_TARGET_VERSION=1.5.x-$BRANCH-SNAPSHOT
...
 # Add-on eXo Remote Edit
-ADDON_REMOTE_EDIT_ORIGIN_VERSION=1.3.x-SNAPSHOT
-ADDON_REMOTE_EDIT_TARGET_VERSION=1.3.x-$BRANCH-SNAPSHOT
+ADDON_REMOTE_EDIT_ORIGIN_VERSION=2.0.x-SNAPSHOT
+ADDON_REMOTE_EDIT_TARGET_VERSION=2.0.x-$BRANCH-SNAPSHOT
```
Comment (vsellier) [inline git-workflow/createFB.sh]: should be 2.1.x-... for both

File: git-workflow/createFB.sh [hunk 6]
```diff
@@ -18,20 +18,20 @@ GATEIN_DEP_TARGET_VERSION=1.5.x-$BRANCH-SNAPSHOT
...
 # Add-on eXo Web Pack
-ADDON_WEB_PACK_ORIGIN_VERSION=1.2.x-SNAPSHOT
-ADDON_WEB_PACK_TARGET_VERSION=1.2.x-$BRANCH-SNAPSHOT
+ADDON_WEB_PACK_ORIGIN_VERSION=2.0.x-SNAPSHOT
+ADDON_WEB_PACK_TARGET_VERSION=2.0.x-$BRANCH-SNAPSHOT
```
Comment (vsellier) [inline git-workflow/createFB.sh]: should be 2.1.x-... for both

File: git-workflow/createFB.sh [hunk 7]
```diff
@@ -18,20 +18,20 @@ GATEIN_DEP_TARGET_VERSION=1.5.x-$BRANCH-SNAPSHOT
...
 # Add-on eXo Web Conferencing
 ADDON_WEB_CONFERENCING_ORIGIN_VERSION=1.1.x-SNAPSHOT
 ADDON_WEB_CONFERENCING_TARGET_VERSION=1.1.x-$BRANCH-SNAPSHOT
```
Comment (vsellier) [inline git-workflow/createFB.sh]: should be 1.2.x-... for both

**Comment by mhbensghaier** [comment]
all addons version has been modified as requested

**Pull request merged**